docs/src/pages/ui/theme/alternativeStyling/index.page.mdx

```diff
@@ -0,0 +1,501 @@
+</Button>;
+```
+
+In the example above, the color of the button will be set to `white`, because the primary variation of the button has a color set to `white` and it takes precedence over the JSS styling. A list of Amplify variations can be found on [here]().
+
+##### Amplify Custom ClassNames
+
+These are custom classnames added onto Amplify components and can be used with CSS styling rules to modify the look of an Amplify component.
+
+```jsx
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { View } from '@aws-amplify/ui-react';
+
+const useStyles = createUseStyles({
+  styledText: {
+    color: 'blue',
+  },
+});
+
+const classes = useStyles();
+const classNames = `${classes.styledText} my-styled-text`;
+
+<Text className={classNames}>This is my JSS Text </Text>;
+```
+
+```CSS
+/* External Style Sheet */
+.my-styled-text {
+  color: red;
+}
+```
+
+In the example above, the color of the text will be `blue` because the JSS styling will take precedence over the classname CSS.
+
+##### CSS Root Variable Overrides
+
+Amplify theming provides many CSS variables to customize the look and feel of the entire application. These CSS variables can be used at a root level to override all instances of component styling.
+
+```jsx
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { Button } from '@aws-amplify/ui-react';
+
+const useStyles = createUseStyles({
+  styledButton: {
+    color: 'blue',
+  },
+});
+
+const classes = useStyles();
+
+<Button className={classes.styledButton}>Button</Button>;
+```
+
+```CSS
+/* External Style Sheet */
+:root {
+  --amplify-components-button-color: red;
+}
+```
+
+In the example above, the button color will be set to `blue` because the JSS styling will take precedence over the Amplify CSS variable.
+
+</Block>
+
+<Block name="Emotion">
+[Emotion](https://github.com/emotion-js/emotion) is a performant and flexible CSS-in-JS library. It allows you to style apps quickly with string or object styles.
+
+#### Usage
+
+```jsx
+/** @jsxImportSource @emotion/react */
+import { jsx, css } from '@emotion/react';
+import { Text } from '@aws-amplify/ui-react';
+
+const styles = {
+  color: 'blue',
+};
+
+//use the Emotion css function and css prop
+<Text css={css(styles)}>This is my Emotion Text </Text>;
+```
+
+#### Interactions
+
+There are various ways to customize an Amplify component and Emotion will interact with these customizations in the following ways:
+
+##### Amplify Styling Props
+
+These are styling props that can be used directly on an Amplify component which will affect a single style property. Examples of styling props are `color` or `fontWeight`. These Amplify styling props will take precedence over Emotion styling.
+
+```jsx
+/** @jsxImportSource @emotion/react */
+import { jsx, css } from '@emotion/react';
+import { Text } from '@aws-amplify/ui-react';
+
+const styles = {
+  color: 'blue',
+};
+
+<Text css={css(styles)} color="red">
```

Comment:
The `css` prop works?? Cool. 

![cool](https://media.giphy.com/media/62PP2yEIAZF6g/giphy.gif)